### PR TITLE
fix compilation

### PR DESCRIPTION
**Pull request by cielavenir**
checked compilation on stretch and bullseye machines.

### CODE REVIEW

File: plugins/rplanners/piecewisepolynomials/openravepy_piecewisepolynomials.cpp [hunk 1]
```diff
@@ -1213,7 +1215,7 @@ OPENRAVE_PYTHON_MODULE(openravepy_piecewisepolynomials)
     .def("Evald1", &PyPiecewisePolynomialTrajectory::Evald1, PY_ARGS("t") "Evaluate the first derivative of this trajectory at the given parameter t")
     .def("Evald2", &PyPiecewisePolynomialTrajectory::Evald2, PY_ARGS("t") "Evaluate the second derivative of this trajectory at the given parameter t")
     .def("Evald3", &PyPiecewisePolynomialTrajectory::Evald3, PY_ARGS("t") "Evaluate the third derivative of this trajectory at the given parameter t")
-    .def("Evaldn", &PyPiecewisePolynomialTrajectory::Evald3, PY_ARGS("t", "n") "Evaluate the n-th derivative of this trajectory at the given parameter t")
+    .def("Evaldn", &PyPiecewisePolynomialTrajectory::Evaldn, PY_ARGS("t", "n") "Evaluate the n-th derivative of this trajectory at the given parameter t")
```
Comment (cielavenir) [inline plugins/rplanners/piecewisepolynomials/openravepy_piecewisepolynomials.cpp:1218]: this one is severe as the signature is different, why did it compile on boost::python?

File: plugins/rplanners/piecewisepolynomials/openravepy_piecewisepolynomials.cpp [hunk 2]
```diff
@@ -35,6 +35,8 @@ namespace py = pybind11;
 namespace py = boost::python;
 #endif // USE_PYBIND11_PYTHON_BINDINGS
 
+#include <openravepy/openravepy_int.h>
+#include <openravepy/openravepy_module.h>
```
Comment (cielavenir) [inline plugins/rplanners/piecewisepolynomials/openravepy_piecewisepolynomials.cpp:39]: without these py::extract is not available on pybind11

**Comment by cielavenir** [comment]
found another issue in boost::python exception, repr(e) was not working as expected. I think this issue existed since the very initial version of openravepy_ext.py.

**Comment by cielavenir** [comment]
> repr(e)

test code https://git.mujin.co.jp/test/test-townsquare/-/issues/1431#note_269290

**Comment by rdiankov** [comment]
thanks!

**Pull request merged**